packages/walletlink/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@web3-react/walletlink",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "8.0.2-alpha.0",
+  "type": "module",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
```

update for consistency based on latest `main`
```suggestion
  "module": "./dist/index.js",
  "types": "./dist/index.d.ts",
```